src/main/java/seedu/myledger/MyLedger.java

```diff
@@ -28,8 +28,8 @@ public static void main(String[] args) {
 
     public static void runMyLedger() {
         start();
-        initializeList();
         Ui.greetUser();
+        initializeList();
```

Comment:
Good placement of the method, this allows any reading corruption error messages to be displayed after the main application greeting for better user interface!